bindings/api.go

```diff
@@ -100,6 +100,15 @@ func GetFundStatus(notificationToken string) ([]byte, error) {
 	return marshalResponse(breez.GetFundStatus(notificationToken))
 }
 
+/*
+RemoveFund is part of the binding inteface which is delegated to breez.RemoveFund
```

Comment:
Nitpick: it's delegated to Remove.Fund**s**

Reply:
thanks, fixed.